core/src/main/java/by/gomel/epam/core/validation/ValidationError.java

```diff
@@ -0,0 +1,10 @@
+package by.gomel.epam.core.validation;
+
+public class ValidationError extends Exception{
+    public ValidationError(String message) {
+        super(message);
+    }
+    public int getStatus(){
+        return 422;
```

Comment:
In this case better to put number under private constant to improve readability of code.